### ISSUE TITLE
Behead

### ISSUE THREAD

**Issue by jamievicary** [homotopy-io/homotopy-rs#172]
New "behead" UI control in the lower-left cluster.

When viewing the nth regular slice of a diagram, selecting "behead" discards all the later slices. (If you are viewing a deeper slice, "behead" should not be operational.)

Alternatively, when viewing the entire diagram in projection, with the mouse cursor hovering over a chevron, choosing "behead" (with the shortcut key) should discard all the later slices above the selected slice.

This lets you "cut off" some part of the tail of your proof, it is very useful in practice.

**Comment by calintat** [comment]
I was bored so I implemented this. I made it so that beheading is only operational if:
1. the diagram is of dimension > 0 and size > 0, and
2. you're either viewing or highlighting (by hovering over a chevron) a boundary or regular slice.

Beheading will then discard all later slices and immediately show you the resulting diagram (if you were viewing a slice).

It's a tad confusing that the behead button only appears if you're hovering over a chevron but then if you try to move your cursor over to the button, it disappears. I contemplated adding a third option so that if you behead a diagram while *not* viewing or highlighting a slice, it just discards the top slice like the old homotopy.io did. Any thoughts?
